README.md

```diff
@@ -111,9 +76,54 @@ TARGET_IMAGE=my-own-repo/testing:latest \
   - If set to `continue`, PrestaShop Flashlight will continue the boot process even if an init script failed
   - Default to `fail`
 
-## Api calls within a docker network
+# Develop
+
+## Build
+
+Requirements:
+
+- [jq](https://jqlang.github.io/jq/)
+
+To build the latest PrestaShop version, simply:
+
+```sh
+./build.sh
+```
+
+For a custom multiplatform build & push:
+
+```sh
+PS_VERSION=8.1.0 \
+PLATFORM=linux/amd64,linux/arm64 \
+PUSH=true \
+TARGET_IMAGE=my-own-repo/testing:latest \
+./build.sh
+```
+
+The `OS_FLAVOUR` defaults to `alpine` (see [Alpine Linux](https://www.alpinelinux.org/)) and `SERVER_FLAVOUR` to `nginx` (see [Nginx](https://www.nginx.com/)).
```

Comment:
It might be a good idea to make a table with the list of variables, their explanations & their default values.

Reply:
You are absolutely right. I will do so